calcom/cal.com
# Only show smsReminderNumber when workflow has ATTENDEE_NUMBER as action

## What does this PR do?

This PR makes sure that the booking question _Phone number (SMS notifications)_ is only show when the event type is active on an workflow that has the action _send SMS to attendee_

Fixes: #7640

## Type of change


## How should this be tested?
Try the following things and check if _Phone number (SMS notifications)_ is correctly shown or not shown as a booking question
- Adding, editing and deleting steps 
- Setting and removing active event type 
- Active/deactivate workflows in `event-types/[id]?tabName=workflows` settings
- Deleting a workflow
